code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -709,18 +710,26 @@ var/global/list/damage_icon_parts = list()
 		else
 			standing.icon = 'icons/mob/belt.dmi'
 
-		if(belt.contents.len && istype(belt, /obj/item/weapon/storage/belt))
-			for(var/obj/item/i in belt.contents)
-				var/i_state = i.item_state
-				if(!i_state) i_state = i.icon_state
-				standing.overlays	+= image("icon" = 'icons/mob/belt.dmi', "icon_state" = "[i_state]")
-
-		overlays_standing[BELT_LAYER] = standing
+		var/belt_layer = BELT_LAYER
+		if(istype(belt, /obj/item/weapon/storage/belt))
+			var/obj/item/weapon/storage/belt/ubelt = belt
+			if(ubelt.show_above_suit)
+				overlays_standing[BELT_LAYER] = null
+				belt_layer = BELT_LAYER_ALT
+			else
+				overlays_standing[BELT_LAYER_ALT] = null
+			if(belt.contents.len)
+				for(var/obj/item/i in belt.contents)
+					var/i_state = i.item_state
+					if(!i_state) i_state = i.icon_state
+					standing.overlays	+= image("icon" = 'icons/mob/belt.dmi', "icon_state" = "[i_state]")

```

Comment:
Yes, it works fine. 

![beltmeme](https://cloud.githubusercontent.com/assets/11579997/17645230/015cf524-6176-11e6-8fca-a629667e2e46.png)